cmd/mt-parrot/generate.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"time"
+
+	"github.com/grafana/metrictank/schema"
+)
+
+func produceArtificialMetrics(schemas []*schema.MetricData) {
+	for tick := range time.NewTicker(artificialMetricsInterval).C {
```

Comment:
this may skip ticks (if our execution of the loop takes too long, e.g. due to a flush taking too long)
see `clock.AlignedTick` for a non-dropping ticker.

Reply:
i'll publish a PR that will help with this, shortly

Reply:
https://github.com/grafana/metrictank/pull/1689

Reply:
updated to use this in cdc44644ea305021e7bcbb36caf1ee68031dff78